README.md

```diff
@@ -0,0 +1,36 @@
+# pyvo-bot
+
+A bot 🤖 which reminds people that the [Pyvo](https://pyvo.cz) meetup 🍻 is coming! 
+
+## Usage
+
+Currently, the bot supports only [Telegram](https://telegram.org/) and can be found at https://t.me/pyvo_bot. The bot runs once per day here using the CircleCI build. It loads iCalendar feed of Pyvo Praha and checks when is the next meetup. If it's about to be soon, it notifies the [Pyvo Praha](https://t.me/pyvopraha) Telegram channel.
+
+## Development
+
+Initial setup:
+```bash
+python3 -m venv venv
+. venv/bin/activate
+pip install -r requirements.txt
+```
+
+Run the code:
+```bash
+python3 main.py 
+```
+
+## To Do
+
+* refactor code (functions could be more decoupled)
+* write tests
+
+Contributions are welcomed. 
```

Comment:
```suggestion
Contributions are welcome. 
```

Reply:
Not an English expert, but I think this is more correct?

Reply:
You're right! Both are grammatically correct and commonly used, but `welcome` is apparently more suitable for this situation.  

https://english.stackexchange.com/questions/2121/which-is-correct-feedback-is-welcome-or-feedback-is-welcomed